.github/workflows/build.yml

```diff
@@ -39,18 +40,30 @@ jobs:
         with:
           submodules: true
 
-      - name: compile
-        uses: KSPModdingLibs/KSPBuildTools/.github/actions/compile@main
+      - uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: 5.x
+
+      # Install CKAN and set up an instance. If CKAN is not needed, remove this step
+      - uses: KSPModdingLibs/KSPBuildTools/.github/actions/ckan-setup@main
```

Comment:
isn't it setup-ckan?  I don't have a strong preference either way, but I think this is currently broken.